kuma/static/js/auth.js

```diff
@@ -54,71 +54,75 @@
+                }
 
-            // After seeing the notice, the user tried to sign in with a second
-            // social account, but was again directed to the registration page.
-            // This will happen if the second social account was also not
-            // tied to an MDN profile.
-            else if(!matchCurrent && matchStored && $browserRegister.length) {
-                localStorage.removeItem(matchKey);
             }
-
-            // After seeing the notice, the user took an action that resulted in
-            // them landing on another page. Either the user abandoned the
-            // registration process or the user tried to sign in with a second
-            // social account and was successful. In either case, the
-            // localStorage item should be immediately removed.
-            else if(!matchCurrent && matchStored) {
-                $doc.on('mdn:login', function(service) {
-                    mdn.Notifier.growl('You can now use ' + matchStored + ' to sign in to this MDN profile.', { duration: 0, closable: true }).success();
-                });
-                localStorage.removeItem(matchKey);
+            catch (e) {
+                // Browser probably doesn't support localStorage
             }
-
-        }
-        catch (e) {
-            // Browser probably doesn't support localStorage
-        }
-    })();
+        })();
+    }
 
     /*
         Fire off events when the user logs in and logs out.
     */
-    if(win.mdn.features.localStorage) (function() {
-        var serviceKey = 'login-service';
-        var serviceStored = localStorage.getItem(serviceKey);
-        var serviceCurrent = $(doc.body).data(serviceKey);
+    if(win.mdn.features.localStorage) {
+        (function() {
+            var serviceKey = 'login-service';
+            var serviceStored = localStorage.getItem(serviceKey);
+            var serviceCurrent = $(doc.body).data(serviceKey);
+
+            try {
+
+                // User just logged in
+                if(serviceCurrent && !serviceStored) {
+                    localStorage.setItem(serviceKey, serviceCurrent);
+                    $doc.trigger('mdn:login', [ serviceCurrent ]);
+                }
+
+                // User just logged out
+                else if(!serviceCurrent && serviceStored) {
+                    localStorage.removeItem(serviceKey);
+                    $doc.trigger('mdn:logout', [ serviceStored ]);
+                }
 
-        try {
-
-            // User just logged in
-            if(serviceCurrent && !serviceStored) {
-                localStorage.setItem(serviceKey, serviceCurrent);
-                $doc.trigger('mdn:login', [ serviceCurrent ]);
             }
-
-            // User just logged out
-            else if(!serviceCurrent && serviceStored) {
-                localStorage.removeItem(serviceKey);
-                $doc.trigger('mdn:logout', [ serviceStored ]);
+            catch (e) {
```

Comment:
Nit: unused parameter `e`

Reply:
I think if we don't use it it will bubble up. The linter did not object.